langchain4j/src/test/java/dev/langchain4j/service/AiServicesIT.java

```diff
@@ -243,12 +243,12 @@ void test_extract_single_enum_with_description() {
         assertThat(weather).isEqualTo(Weather.RAINY);
 
         verify(chatLanguageModel).generate(singletonList(userMessage("Analyze weather forecast for:\n" +
-                "|||" + weatherForecast + "|||\n" +
```

Comment:
please revert reformatting everywhere in this PR

Reply:
reverted this file. It belongs to another PR #2064

Reply:
Please allow me to push in this PR and [another one](https://github.com/langchain4j/langchain4j/pull/2064), I will revert all reformatting changes myself.
I do not want to merge all these reformatting changes and then re-reformat it back in a separate PR and thus lose some history.

Reply:
I am getting the following error when trying to push in this branch:
```
12:03:09.787: [langchain4j_1] git -c credential.helper= -c core.quotepath=false -c log.showSignature=false push --progress --porcelain kpavlov refs/heads/fork/kpavlov/delombok-lc4j:delombok-lc4j
error: Authentication error: Authentication required: You must have push access to verify locks
error: failed to push some refs to 'https://github.com/kpavlov/langchain4j'
```

Reply:
I sent you an invite to @langchain4j account

Reply:
I've tried to reformat, but idea did no changes. I hope, reformatting all the code in the project (in a separate PR) will fix existing codestyle violations. I am up for strict automatic validation and formatting before push, as it will handle code style issues automatically.